include/emitc/emitc_tosa.h

```diff
@@ -351,6 +351,35 @@ inline Dest reshape(Src x) {
   return emitc::reshape<Dest>(x);
 }
 
+// TransposeOp
+// Maps the perms dimension from Dest to Src
+template <typename Dest, typename Src>
+inline Dest transpose(Src operand, Tensor1D<int64_t, Src::rank()> perms) {
+  static_assert(is_tensor<Src>::value, "Expected tensor argument");
+  static_assert(is_tensor<Dest>::value, "Expected tensor result");
+
+  // Since broadcast_dimensions map the broadcast_dimensions from Src to Dest,
+  // we have to reverse the tosa.transpose perms mapping rom Dest to Src
```

Comment:
```suggestion
  // we have to reverse the tosa.transpose perms mapping from Dest to Src
```
typo

Reply:
You're right, but the comment was updated anyway